### PR TITLE
Removed print of token/password in logs

### PR DESCRIPTION
**Pull request by Galorhallen**
As per title.
Removed a log line that was printing the api token or password.
The same log now just print the field name (if value is not empty) and the value is redacted.

### CODE REVIEW

File: config/configuration.go [hunk 1]
```diff
@@ -78,7 +78,13 @@ func (c *Config) String() string {
...
 		valueField := fields.Field(i)
 		typeField := fields.Type().Field(i)
-		buffer[i] = fmt.Sprintf("%s=%v", typeField.Name, valueField.Interface())
+		if typeField.Name != "PIHolePassword" && typeField.Name != "PIHoleApiToken" {
+			buffer[i] = fmt.Sprintf("%s=%v", typeField.Name, valueField.Interface())
+		} else {
+			if valueField.Len() > 0 {
+				buffer[i] = fmt.Sprintf("%s=%s", typeField.Name, "*****")
+			}
+		}
```
Comment (eko) [inline config/configuration.go:87]:
```suggestion
		} else if valueField.Len() > 0 {
			buffer[i] = fmt.Sprintf("%s=%s", typeField.Name, "*****")
		}
```

File: go.mod [hunk 2]
```diff
@@ -9,19 +9,3 @@ require (
 	github.com/xonvanetta/shutdown v0.0.3
 	golang.org/x/net v0.0.0-20200625001655-4c5254603344
 )
-
```
Comment (eko) [inline go.mod:12]: Could you please keep these requires? I think you are using an old version of Go: you need to upgrade to 1.17 to keep them.

**Review by eko — COMMENTED** [review]
Hi @Galorhallen,

Thank you for your pull request! I'm looking forward to see it merged.

I just leaved two little comments if you have a couple of minutes to fix them. Elsewhere don't hesitate to let me know.

File: config/configuration.go (hunk 1 above)
Comment (Galorhallen) [reply config/configuration.go]: Fixed

File: go.mod (hunk 2 above)
Comment (Galorhallen) [reply go.mod]: Fixed.
I did the last changes on WSL, without noticing that it had a *very* old go version

**Review by eko — APPROVED** [review]
👍

**Pull request merged**